.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+	"extends": ["airbnb", "plugin:prettier/recommended", "plugin:jest/recommended"],
+	"rules": {
+		"react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+		"class-methods-use-this": 0,
+		"react/prefer-stateless-function": 0,
+		"react/no-did-update-set-state": 0,
+		"import/no-unresolved": 0,
```

Comment:
Are these rules necessary since we don't have react around here?

Reply:
I changed the airbnb dependency to airbnb-base and removed all dependencies related to react